<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMenu</class>
 <widget class="QDialog" name="ClientMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="seeProductsButton">
      <property name="text">
       <string>Просмотр товаров</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buscketButton">
      <property name="text">
       <string>Корзина</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ordersButton">
      <property name="text">
       <string>Мои заказы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>ClientMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
